yarn-project/p2p-bootstrap/src/index.ts

```diff
@@ -18,7 +19,18 @@ async function main(
   telemetryClient: TelemetryClient = getTelemetryClient(),
   logger = debugLogger,
 ) {
-  const store = await createStore('p2p-bootstrap', 1, config, logger);
+  const store = await createStore(
+    'p2p-bootstrap',
+    1,
+    {
+      dataStoreMapSizeKB: config.dataStoreMapSizeKB,
+      dataDirectory: config.dataDirectory,
+      l1Contracts: {
+        rollupAddress: EthAddress.ZERO,
```

Comment:
Im not sure we always want this to be zero, it will cause version mismatches in the boot nodes. 

Will check and update this comment